raiden_contracts/utils/mint_tokens.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import click
+
+from eth_utils import encode_hex
+from web3 import HTTPProvider, Web3
+from web3.middleware import construct_sign_and_send_raw_middleware
+from raiden_contracts.utils.private_key import get_private_key
+from raiden_contracts.utils.signature import private_key_to_address
+from raiden_contracts.contract_manager import ContractManager, contracts_precompiled_path
+from raiden_contracts.constants import CONTRACT_CUSTOM_TOKEN
+
+WEI_TO_ETH = 10 ** 18
```

Comment:
nitpick: this constant is never used.

Reply:
Sorry this was not caught by flake8 when i ran it.